packages/experimental/ExperimentalIcon/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@fluentui-react-native/experimental-icon",
+  "version": "0.0.1",
+  "description": "A cross-platform Icon component",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "clean": "fluentui-scripts clean",
+    "depcheck": "fluentui-scripts depcheck",
+    "just": "fluentui-scripts",
+    "lint": "fluentui-scripts eslint",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui-react-native.git",
+    "directory": "packages/experimental/ExperimentalIcon"
+  },
+  "dependencies": {
+    "@fluentui-react-native/adapters": "^0.9.3",
```

Comment:
Just a heads up that these versions are not aligned with what exists in the repo at the moment